packages/typespec-ts/src/modelUtils.ts

```diff
@@ -54,6 +60,11 @@ import {
 } from "@typespec/http";
 import { getPagedResult, isFixed } from "@azure-tools/typespec-azure-core";
 import { extractPagedMetadataNested } from "./operationUtil.js";
+import {
+  DpgContext,
+  getDefaultApiVersion,
```

Comment:
I remember we need to do something in the tcgc about the getDefaultApiVersion logic ? Has that been done ?